README.md

```diff
@@ -6,8 +6,53 @@ How to start the floorIsJava application
 1. Run `mvn clean install` to build your application
 1. Start application with `java -jar target/FloorIsJava-1.0-SNAPSHOT.jar server config.yml`
 1. To check that your application is running enter url `http://localhost:8080`
+2. To see your application's Swagger UI http://localhost:8080/swagger
 
-Health Check
+Database
 ---
+Create an empty database: 
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
+Run the following command from the base of your cloned directory to create the required database structure:
+`mysql --host=<localhost> --user=<your_username> --password=<your_password> <your_database_name> < employeesdb.sql`
```

Comment:
do you have a sql file called employeesdb.sql? if not this line won't work

Reply:
reviewed and updated